src/ensembl/src/content/app/entity-viewer/shared/components/entity-viewer-sidebar/entity-viewer-sidebar-modal/modal-views/EntityViewerBookmarks.tsx

```diff
@@ -47,6 +48,11 @@ type PreviouslyViewedLinksProps = {
 export const PreviouslyViewedLinks = (props: PreviouslyViewedLinksProps) => {
   const dispatch = useDispatch();
 
+  const onClick = () => {
```

Comment:
minor, should we call this handleClick() or name as per the functionality?

Reply:
I am not sure what we do in 2020 code, i dont see any problem with using either. Lets see what others say.

Reply:
@jyothishnt this is not relevant anymore because I have changed the way we are doing this. Please have a look at the changes again.